python/cuspatial/cuspatial/core/gis.py

```diff
@@ -107,21 +119,28 @@ def haversine_distance(p1_lon, p1_lat, p2_lon, p2_lat):
 
 
 def lonlat_to_cartesian(origin_lon, origin_lat, input_lon, input_lat):
-    """ Convert lonlat coordinates to km x,y coordinates based on some camera
-    origin.
-
-    params
-    origin_lon: float64 - longitude camera
-    origin_lat: float64 - latitude camera
-    input_lon: Series of longitude coords to convert to x
-    input_lat: Series of latitude coords to convert to y
+    """
+    Convert lon/lat coordinates to kilometer x,y coordinates with respect to
+    an origin camera position.
```

Comment:
Convert lon/lat to x,y coordinates relative to an origin lon/lat. Results are scaled relative to the size of Earth in kilometers.

Reply:
please make a github suggestion as to what to change?